src/main/scala/com/lightbend/play/spring/SpringBuilder.scala

```diff
@@ -193,6 +195,11 @@ abstract class SpringBuilder[Self] protected (
     ctx.refresh()
     ctx.start()
 
+    injector.instanceOf[ApplicationLifecycle].addStopHook { () =>
+      ctx.asInstanceOf[ConfigurableApplicationContext].close()
```

Comment:
any try catch?

Reply:
Is this cast needed? `AnnotationConfigApplicationContext` [inherits from `org.springframework.context.support.AbstractApplicationContext` which has a `close` method](https://docs.spring.io/spring/docs/4.3.11.RELEASE/javadoc-api/org/springframework/context/annotation/AnnotationConfigApplicationContext.html#methods.inherited.from.class.org.springframework.context.support.AbstractApplicationContext).

Reply:
I'll look at it this weekend